src/Model/GroundWaterTransport/gwt1cnc1.f90

```diff
@@ -160,36 +172,42 @@ subroutine cnc_rp(this)
     !
     ! -- Stop if errors detected
     if (ierr > 0) then
-      call this%parser%StoreErrorUnit()
+      call store_error_filename(this%input_fname)
     end if
     !
-    ! -- return
+    ! -- Write the list to iout if requested
+    if (this%iprpak /= 0) then
+      call this%write_list()
+    end if
+    !
+    ! -- Return
     return
   end subroutine cnc_rp
 
+  !> @brief Constant concentration/temperature package advance routine
+  !!
+  !! Add package connections to matrix
+  !<
   subroutine cnc_ad(this)
-! ******************************************************************************
-! cnc_ad -- Advance
-! ******************************************************************************
-!
-!    SPECIFICATIONS:
-! ------------------------------------------------------------------------------
     ! -- modules
     ! -- dummy
     class(GwtCncType) :: this
     ! -- local
     integer(I4B) :: i, node
     real(DP) :: cb
     ! -- formats
-! ------------------------------------------------------------------------------
     !
     ! -- Advance the time series
     call this%TsManager%ad()
     !
     ! -- Process each entry in the constant concentration cell list
     do i = 1, this%nbound
       node = this%nodelist(i)
-      cb = this%bound(1, i)
+      if (this%iauxmultcol > 0) then
+        cb = this%tspvar(i) * this%auxvar(this%iauxmultcol, i)
+      else
+        cb = this%tspvar(i)
+      end if
```

Comment:
Shouldn't this be replaced with `cb = conc_mult(i)`

Reply:
addressed in a12332c